source/features/update-pr-from-base-branch.tsx

```diff
@@ -51,9 +51,27 @@ async function handler({delegateTarget}: delegate.Event): Promise<void> {
 
 async function addButton(position: Element): Promise<void> {
 	const {base, head} = getBranches();
-	const {status} = await api.v3(`compare/${base}...${head}`);
+	const [comparison, deepData] = await Promise.all([
+		// TODO: Find how to determine whether the branch needs to be updated via v4
+		api.v3(`compare/${base}...${head}`),
 
-	if (status === 'diverged') {
+		// Some fields are just for testing
+		api.v4(`
```

Comment:
Ideally we'd just use v4, but I still don't see any fields for this. https://twitter.com/fregante/status/1543557018576830464